tests/catalog/test_models.py

```diff
@@ -289,3 +289,19 @@ def test_long_name(self):
             self.assertLessEqual(len(tag.slug), catalog.models.SLUG_MAX_LENGTH)
         except DataError as e:
             self.assertTrue(False, f'Tag has too long name. {e}')
+
+    def test_order_by_alphanumeric(self):
+        ordered_tags = [
+            catalog_models.MockTag(name='a'),
+            catalog_models.MockTag(name='b'),
+            catalog_models.MockTag(name='1.2 В'),
+            catalog_models.MockTag(name='1.6 В'),
+            catalog_models.MockTag(name='5 В'),
+            catalog_models.MockTag(name='12 В'),
+        ]
```

Comment:
it should be good to add numeric values with another dimension: `['1.2 A', '6 A']` for example